datasaur-ai/robosaur
# #11979 Add tag filter for export project

**Changes**
- Add tags to projectFilter export
- Add projectFilter.tags (tags name) to tagIds converter

**How to test**
- Try to export project with tags => it will export tagged project
- Try to export project with wrong tags => it will throw error tags not found
- Try to export project with no projectFilter.tag => make sure existing behavior is fine
